modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/CreateDumpFutureTask.java

```diff
@@ -708,7 +714,12 @@ private boolean writtenByIterator(int cache, KeyCacheObject key, CacheObjectCont
             while (writers.get() > 0) // Waiting for all on the fly listeners to complete.
                 LockSupport.parkNanos(1_000_000);
 
-            U.closeQuiet(file);
+            try {
+                file.close();
+            }
+            catch (IOException e) {
+                throw new IgniteException(e);
```

Comment:
Do we have a test for that. The test need check that this exception doesn't affect Ignite node, only the ongoing dump process.

Reply:
Yes, it is tested in `IgniteCacheDumpSelfTest#testDumpCancelOnFileCreateError`. Also added extra checks to make sure ignite provides data after the error.